pg_search/src/query/mod.rs

```diff
@@ -592,9 +592,55 @@ fn check_range_bounds(
     let one_day_nanos: i64 = 86_400_000_000_000;
     let lower_bound = match (typeoid, lower_bound.clone()) {
         // Excluded U64 needs to be canonicalized
```

Comment:
Could you add an explanation of what you mean by "canonicalization" in a comment somewhere? It was likely already in the PR description, but having it inline in the code would be helpful.